HW0/P5/P5.txt

```diff
@@ -1 +1,41 @@
 # Your answers here.
```

Comment:
Great, it's good to know that my long explanations were helpful... Thanks for the feedback!